config/config.go

```diff
@@ -33,7 +33,20 @@ import (
 )
 
 var (
-	patRulePath = regexp.MustCompile(`^[^*]*(\*[^/]*)?$`)
+	patRulePath         = regexp.MustCompile(`^[^*]*(\*[^/]*)?$`)
+	unchangeableHeaders = map[string]struct{}{
+		"host":                              {},
+		"authorization":                     {},
```

Comment:
this could be commented (so you know you can't change it by reading the code) with a note saying it is checked elsewhere

Reply:
Commented it out with a note.